src/mono/wasm/runtime/net6-legacy/method-calls.ts

```diff
@@ -296,4 +296,59 @@ export function mono_wasm_invoke_js_blazor(exceptionMessage: Int32Ptr, callInfo:
         exceptionRoot.release();
         return 0;
     }
-}
\ No newline at end of file
+}
+
+export function mono_wasm_change_case_invariant(exceptionMessage: Int32Ptr, src: number, srcLength: number, dst: number, dstLength: number, toUpper: boolean) : void{
```

Comment:
afaik this type info is incorrect, the only types that can cross the wasm boundary are f32, f64, i32 in most cases. so toUpper will be an int with a value of 0 or 1. this is generally harmless but it does mean that if you were to do `toUpper === true` it would not do what you expect, so it's best to use `number` here.